modules/gke-cluster-standard/main.tf

```diff
@@ -409,9 +421,6 @@ resource "google_container_cluster" "cluster" {
         enable_relay = (
           var.monitoring_config.advanced_datapath_observability.enable_relay
         )
-        relay_mode = (
-          var.monitoring_config.advanced_datapath_observability.relay_mode
-        )
```

Comment:
are you removing this intentionally?

Reply:
Yes because it's deprecated and will be removed in a future release: https://github.com/hashicorp/terraform-provider-google/blob/db268a0e399d5da16e0c7ab47a2db6f3ea04894e/google/services/container/resource_container_cluster.go#L1092C1-L1092C17